common/src/main/res/values/strings.xml

```diff
@@ -252,6 +252,7 @@
     <string name="entity_id">Entity ID</string>
     <string name="entity_id_name">Entity ID: %1$s</string>
     <string name="entity_widget_desc">View any entity\'s current state and toggle it</string>
+    <string name="todo_widget_desc">View configured to-do list</string>
```

Comment:
```suggestion
    <string name="todo_widget_desc">View any configured to-do list</string>
```

The other widgets also describe they have actions we could also mention that here such as marking items as completed, my suggestion was just for readability and hopefully help translators :)